src/client/components/index.js

```diff
@@ -2,6 +2,15 @@ import RoomSelector from './RoomSelector';
 import NameSelector from './NameSelector';
 import GameContainer from './GameContainer';
 
+window.addEventListener("orientationchange", (event) => {
+  const angle = Math.abs(event.target.screen.orientation.angle);
+  if(angle === 90 || angle === 270) {
+    document.body.requestFullscreen();
+  } else {
+    document.exitFullscreen();
+  }
+});
+
```

Comment:
> call it once immediately when registering it so that it will work if the page is loaded in landscape mode

Unfortunately this will not work, because there's no `orientationchange` event on page start. Also browsers usually don't allow to perform this kind of actions if there was no input from user. Added it as a separate call just in case :)

Reply:
Oh, is window.orientation just null on page-load? Also you're right, I forgot that fullscreen is one of those things you can't do without user-input. You can just drop that last commit entirely then. Maybe I'll think about having double-tap toggle fullscreen or something for mobile but that wouldn't go through this code anyway since it wouldn't need to check orientation.